.github/workflows/perf-device-models.yaml

```diff
@@ -36,9 +36,10 @@ jobs:
         run: |
           ./scripts/build_scripts/build_with_profiler_opt.sh
       - name: Run device performance regressions
-        timeout-minutes: 60
+        timeout-minutes: 90
```

Comment:
these tests take longer than 60 minutes now?

Reply:
No it is at 55, at some point it did go higher but couple of optimizations brought it to 55 min. Main is at 44 min for this CI.  The real contributor for the difference is bringing GS profiler regression test to this CI run as well.  `test_multi_op`  has been added that will fail if the GS card is resetted using tt-smi. That adds around 8-9 minutes. 55-56 felt too close to 60, so I kept the 90 min timeout.

Reply:
We are trying to keep post-commit under 45 minutes... can we discuss what is absolutely crucial to be on post-commit for profiler, and what we can move to a scheduled pipeline?

Reply:
I believe `Device perf regressions and output report` is a nightly/scheduled call no?  Is it called as part of post-commit and I am missing it? 

If that is is the case, this actually removes the need for GS profiler runs on post commit. Slightly easing requirements on GS machines

Reply:
NVM AM BLIND SRY